Reaqtive/Core/Reaqtive.Linq/Reaqtive/Subscribable.cs

```diff
@@ -630,6 +630,24 @@ public static ISubscribable<TSource> Finally<TSource>(this ISubscribable<TSource
 
         #region Reactive Operators
 
+        #region DefaultIfEmpty
+
+        /// <summary>
+        /// Propagates all elements from the source sequence, or a default item if the source sequence emits nothing.
+        /// </summary>
+        /// <typeparam name="TSource">Type of the elements in the source sequence.</typeparam>
+        /// <param name="source">Source sequence whose elements to propagate.</param>
+        /// <returns>Observable sequence containing all elements from the source sequence, or a default item.</returns>
+        public static ISubscribable<TSource> DefaultIfEmpty<TSource>(this ISubscribable<TSource> source)
```

Comment:
This one is good, but we should add the overload with the `TSource` parameter as well.